torch/_inductor/codegen/mps.py

```diff
@@ -527,9 +543,9 @@ def reduction(
                 dtype=dtype,
             )
         if reduction_type == "welford_reduce":
-            assert not self.multistage_reduction, (
-                f"Multistage reduction not yet supported for {reduction_type}"
-            )
+            assert (
```

Comment:
I really don't understand why there are these unrelated changes. Did something change in lintrunner? Can we consider reverting that?

Reply:
Sorry, those should not have been there, different linter versions

Reply:
No worries, I actually experienced the same locally when adding another op.